src/selectors/per-thread/markers.js

```diff
@@ -186,16 +186,28 @@ export function getMarkerSelectorsPerThread(threadSelectors: *) {
     }
   );
 
+  /**
+   * This selector applies the committed range to the full list of markers.
```

Comment:
I think this code comment could be a bit more accurate, especially around "full list of markers".